tests/notes/class-wc-tests-customizing-product-catalog.php

```diff
@@ -0,0 +1,131 @@
+<?php
```

Comment:
Nice work on these test cases, very thorough 👍 👍

Reply:
@louwie17 thank you!